pkg/manager/client/discovery.go

```diff
@@ -1,10 +1,11 @@
 package client
 
 import (
-	"bytes"
 	"context"
-	"encoding/json"
 	"fmt"
+	"github.com/Jeffail/gabs/v2"
```

Comment:
Move `github.com` import into the second section of the import, along with logrus.